src/+otp/+kortewegdevries/KortewegdeVriesProblem.m

```diff
@@ -0,0 +1,102 @@
+classdef KortewegdeVriesProblem < otp.Problem
+
+    % Ascher, Uri M. and Robert I. McLachlan. “On Symplectic and 
+    % Multisymplectic Schemes for the KdV Equation.” 
+    % Journal of Scientific Computing 25 (2005): 83-104.
+    %
+    % Robert M. Miura, Clifford S. Gardner, Martin D. Kruskal;
+    % Korteweg‐de Vries Equation and Generalizations. II. 
+    % Existence of Conservation Laws and Constants of Motion. 
+    % J. Math. Phys. 1 August 1968; 9 (8): 1204–1209. 
+    % https://doi.org/10.1063/1.1664701
+
+    methods
+        function obj = KortewegdeVriesProblem(timeSpan, y0, parameters)
+            
+            obj@otp.Problem('Korteweg de Vries Equation', [], ...
+                timeSpan, y0, parameters);
+            
+        end
+    end
+    
+    properties (SetAccess = private)
+        DistanceFunction
+        numInvariants
+        Invariant
+        JacobianInvariant
```

Comment:
This needs to be part of an RHS. I would do like RHSInvariant

Reply:
Basically resolved since its not a DAE